Code/client/Games/Skyrim/Forms/LoadedCellData.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
```

Comment:
LoadedCellData is not a form. It is part of TESObjectCELL, so you should move the struct to be within TESObjectCELL.